Modules/Devices/src/Devices.AdminCli/Program.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.Reflection;
 using System.Text.Json;
-using Devices.Domain.Entities;
-using Devices.Infrastructure.Persistence.Database;
+using Backbone.Modules.Devices.Domain.Entities;
+using Backbone.Modules.Devices.Infrastructure.Persistence.Database;
```

Comment:
Shouldn't the Devices.AdminCli project be using a similar prefix in its namespace?
e.g. **Backbone.Modules.Devices.AdminCli**

Reply:
Good catch!